src/Adapters/ADTAdapter.ts

```diff
@@ -473,6 +475,30 @@ export default class ADTAdapter implements IADTAdapter {
         );
     }
 
+    advancedSearchADTTwins(
```

Comment:
can we give it a name that agnostic of where it's used? Maybe `searchTwins` or `executeTwinSearch`?

Reply:
I'll change the name, but I prefer something around `customTwinSearch` or something that talks about a free-form query.

Reply:
hmm, `twinSearchByQuery`?